src/utils/format-time-length.js

```diff
@@ -0,0 +1,20 @@
+// @flow
```

Comment:
Maybe? I'm not sure on what's appropriate there but I love things yelling at me when I'm wrong.